### DISCUSSION TITLE
refresh table's relationship on nocodb after updating and creating foreign keys on table [Apr 27]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1976] [Discord Threads]
> 🙋‍♂️ **BoMBx**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### internally or externally?  
#  
> 👨‍🚀 **wingkwong**  
#### n wt does it mean by "refresh table's relationship on nocodb"  
#  
> 🙋‍♂️ **BoMBx**  
#### I mean I need data of joining table but at initial project on nocodb I didn't create foreign keys yet then I cannot fetch data of joining between two or multiple tables  
#  
> 🙋‍♂️ **BoMBx**  
#### I just did it with external program but it is same database that nocodb connect to  
#  
> 🙋‍♂️ **BoMBx**  
#### If got other way to set foreign keys via nocodb directly would be great.  
#  
> 👨‍🚀 **wingkwong**  
#### it's like LinkToAnotherRecord but with real relation in db?  
#  
> 🙋‍♂️ **BoMBx**  
#### I think I misunderstand about your LinkToAnotherRecord  
#  
> 👨‍🚀 **wingkwong**  
#### if FKs are present, then LinkToAnotherRecord will be created automatically  
#  
> 🙋‍♂️ **BoMBx**  
#### what i mean is .. can it automatically join tables after creating foreign keys as initial project?  
#  
> 🙋‍♂️ **BoMBx**  
#### really? I don't know why my table it not automatically show up  
#  
> 🙋‍♂️ **BoMBx**  
#### let me show you  
#  
> 🙋‍♂️ **BoMBx**  
####   
![image0](https://cdn.discordapp.com/attachments/968358983599525918/968394335219503155/unknown.png)  
#  
> 🙋‍♂️ **BoMBx**  
#### it was set here  
#  
> 🙋‍♂️ **BoMBx**  
####   
![image0](https://cdn.discordapp.com/attachments/968358983599525918/968394677965447239/unknown.png)  
#  
> 🙋‍♂️ **BoMBx**  
#### here is the nocodb  
#  
> 🙋‍♂️ **BoMBx**  
#### seems nothing change  
#  
> 🙋‍♂️ **BoMBx**  
####   
![image0](https://cdn.discordapp.com/attachments/968358983599525918/968396035728764949/unknown.png)  
#  
> 👨‍🚀 **wingkwong**  
#### do they exist before the project creation?  
#  
> 🙋‍♂️ **BoMBx**  
#### nope  
#  
> 🙋‍♂️ **BoMBx**  
#### I just adding today  
#  
> 👨‍🚀 **wingkwong**  
#### if you go to meta page, r u able to see the sync  
#  
> 🙋‍♂️ **BoMBx**  
#### do you have any image of it?  
#  
> 🙋‍♂️ **BoMBx**  
#### I cannot found the sync menu  
#  
> 🙋‍♂️ **BoMBx**  
####   
![image0](https://cdn.discordapp.com/attachments/968358983599525918/968451388654891008/unknown.png)  
#  
> 🙋‍♂️ **BoMBx**  
#### which part of screen?  
#  
> 💁 **o1lab**  
#### if you oepn project   
#### left bottom : Team and settings > Metadata  
#  
> 🙋‍♂️ **BoMBx**  
####   
![image0](https://cdn.discordapp.com/attachments/968358983599525918/968456007745417300/unknown.png)  
#  
> 🙋‍♂️ **BoMBx**  
#### hmmm.. I see. Just press `Sync  Now` button, right?  
#  
> 🙋‍♂️ **BoMBx**  
#### now can use it, really thanks both of you  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/968358983599525918)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
